source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRViewController.mm

```diff
@@ -190,8 +190,9 @@ - (void) addTasksToConcurrentQueue:(std::vector<std::shared_ptr<BaseCardElement>
                         NSString *parsedString = [NSString stringWithCString:markDownParser->TransformToHtml().c_str() encoding:NSUTF8StringEncoding];
 
                         // Font and text size are applied as CSS style by appending it to the html string -- font is hard coded for now
```

Comment:
>-- font is hard coded for now [](start = 106, length = 29)

Update comment

Reply:
thanks!

---
In reply to: [169504564](https://github.com/Microsoft/AdaptiveCards/pull/1145#discussion_r169504564) [](ancestors = 169504564)